helm/test/test_idol_nifi.py

```diff
@@ -141,5 +141,27 @@ def test_registry_buckets(self):
         self.assertEqual(objs['ConfigMap']['idol-nifi-reg-cm']['data']['NIFI_REGISTRY_BUCKET_NAME_2'], 'some-bucket')
         self.assertEqual(objs['ConfigMap']['idol-nifi-reg-cm']['data']['NIFI_REGISTRY_BUCKET_FILES_2'], '/some/flow/file1.json,/some/flow/file2.json')
 
+    def test_additional_volumes(self):
+        objs = self.render_chart(
+            {
+                'additionalVolumeMounts': [
+                    {"name": "test-nifi-files", "mountPath": "/some/mount/point"}
+                ],
+                'additionalVolumes': [
+                    {"name": "test-nifi-files", "configMap": {"name": "test-nifi-cfg-map"}}
+                ]
+            })
+        mounts = objs['StatefulSet']['idol-nifi-reg']['spec']['template']['spec']['containers'][0]['volumeMounts']
+        self.assertEqual(7, len(mounts))
+        additional_mount = mounts[-1]
+        self.assertEqual("test-nifi-files", additional_mount["name"])
+        self.assertEqual("/some/mount/point", additional_mount["mountPath"])
+
+        volumes = objs['StatefulSet']['idol-nifi-reg']['spec']['template']['spec']['volumes']
+        self.assertEqual(3, len(volumes))
+        additional_volume = volumes[-1]
+        self.assertEqual("test-nifi-files", additional_volume["name"])
+        self.assertEqual("test-nifi-cfg-map", additional_volume["configMap"]["name"])
```

Comment:
You could just check that the additional entry is somewhere in the array, rather than at a particular index

Just thinking might be possible to generalize this test to the rest of the idol- charts that support `additionalVolumes` and `additionalVolumeMounts`